### PR TITLE
Add the ability to generate appup files using bundle libs

### PR DESCRIPTION
**Pull request by joaohf**
This allows the appups to be created when the libs are
in this generic layout:

    bundle/
        apps/
            app1
            app2
            app3

Each appX is a standard OTP application.

Building:

    rebar3 compile

    find -name *.appup

    _build/default/lib/app1/ebin/app1.appup
    _build/default/lib/app3/ebin/app3.appup
    _build/default/lib/myapp/ebin/myapp.appup
    _build/default/lib/app2/ebin/app2.appup

### CODE REVIEW

**Comment by joaohf** [comment]
Hi,

Using rebar3_appup_plugin patched

rebar3 compile

find -name *.appup

./appup_bundle/_build/default/lib/app1/ebin/app1.appup
./appup_bundle/_build/default/lib/app3/ebin/app3.appup
./appup_bundle/_build/default/lib/myapp/ebin/myapp.appup
./appup_bundle/_build/default/lib/app2/ebin/app2.appup

Using rebar3_appup_plugin from hex.pm

rebar3 compile
find -name *.appup

This PR is the behaviour that I've implemented  in https://github.com/joaohf/rebar3_appup.

The files src/*.appup.src should be in _build/default/lib/app/ebin/*.appup after the compile. Even using any kind of layout like "bundle".

Thanks.

File: src/rebar3_appup_compile.erl [hunk 1]
```diff
@@ -35,13 +35,15 @@ do(State) ->
...
-                                end)
+        SrcDir = appup_file_src(AppInfo),
+	TargetDir = appup_file_target(AppInfo),
+	compile(SrcDir, TargetDir, Opts)
+%	rebar_base_compiler:run(Opts, [],
+%                                SrcDirs, ".appup.src",
+%                                TargetDir, ".appup",
+%                                fun(Source, Target, Config) ->
+%                                    compile(Source, Target, Config)
+%                                end)
```
Comment (lrascao) [inline src/rebar3_appup_compile.erl]: This commented code can be removed

**Comment by lrascao** [comment]
Thanks!
I've added a small note on the commit to remove commented out code, also could you please make the PR target to `develop` instead of `master`? I'll cut a new release/tag with your fix

**Comment by joaohf** [comment]
Done. See #3

**Pull request closed without merging**